tools/windows-node-installer/README.md

```diff
@@ -46,39 +45,43 @@ Flags:
+worker node.
 
 Usage:
-  wni aws [command]
-
-Available Commands:
-  create      Create a Windows instance on the same provider as the existing OpenShift Cluster.
-  destroy     Destroy the Windows instances and resources specified in 'windows-node-installer.json' file.
+  wni aws create [flags]
 
 Flags:
-      --credential-account string   account name of a credential used to create the OpenShift Cluster specified in the provider's credentials file. (required)
-      --credentials string          file path to the cloud provider credentials of the existing OpenShift cluster (required).
-  -h, --help                        help for aws
+  -h, --help                   help for create
+      --image-id string        ami ID of a base image for the instance (i.e.: ami-06a4e829b8bbad61e for Microsoft 
+      Windows Server 2019 Base image on AWS). (required)
+      --instance-type string   name of a type of instance (i.e.: m4.large for AWS, etc). (required)
+      --ssh-key string         name of existing ssh key on cloud provider for accessing the instance after it is 
+      created. (required)
 
 Global Flags:
-      --dir string          directory to save or read window-node-installer.json file from. (default ".")
-      --kubeconfig string   file path to the kubeconfig of the existing OpenShift cluster (required).
-      --log-level string    log level (e.g. 'info') (default "info")
+      --credential-account string   account name of a credential used to create the OpenShift Cluster specified in 
+      the provider's credentials file. (required)
+      --credentials string          file path to the cloud provider credentials of the existing OpenShift cluster 
+      (required).
+      --dir string                  directory to save or read window-node-installer.json file from. (default ".")
+      --kubeconfig string           file path to the kubeconfig of the existing OpenShift cluster (required).
+      --log-level string            log level (e.g. 'info') (default "info")
 
-Use "wni aws [command] --help" for more information about a command.
 ```
 
 Created instance default properties:
- - Instance name \<OpenShift cluster infrastructure ID\>-windows-worker-\<zone\>-\<random 4 characters string\>
- - Share virtual network with the OpenShift Cluster
- - Public subnet within the virtual network
+ - Instance name <OpenShift cluster\'s infrastructure ID>-windows-worker-\<zone\>-<random 4 characters string>
+ - Sharing virtual network with the OpenShift Cluster
+ - Using public subnet within the virtual network
  - Auto-assigned public IP address
- - Attached security group for Windows (Allow RDP access from user's IP address and all traffic within the virtual 
- network)
+ - Attached with a security group for Windows (Allow RDP access from user\'s IP address and all traffic within the 
+ virtual network)
  - Attached OpenShift cluster worker IAM role (Profile)
  - Attached OpenShift cluster worker security group
```

Comment:
`Attached with the OpenShift cluster's worker security group`. Please move this before the IAM bullet.